src/common/http_header.cc

```diff
@@ -283,6 +281,10 @@ Connection::write(std::ostream& os) const {
     case ConnectionControl::KeepAlive:
         os << "Keep-Alive";
         break;
+    case ConnectionControl::Ext:
```

Comment:
Right. I'm actually not very happy with the current handling of extensions in various headers. I think I'll try to use `std::variant` to represent them and have a special `Extension` struct